### ISSUE TITLE
Windows installer dialog is tiny on high-resolution displays

### ISSUE THREAD

**Issue by johann-petrak** [GateNLP/gate-core#99]
I downloaded the Windows installer for 8.6 from the website to my Windows 10 laptop with 3200x1800 resolution and the dialog window of the installer and especially the license text and scrollbar for it are just tiny. 
See attached screen shot. 
![Screenshot (1)](https://user-images.githubusercontent.com/619106/71191116-b881eb80-227d-11ea-85e6-e80bbd0a33d7.png)

**Comment by johann-petrak** [comment]
The 8.7-SNAPSHOT installer as of today looks the same.

**Comment by ianroberts** [comment]
What version of Java are you using?  Anything before 9 doesn't handle hi-DPI monitors at all well (OK, it does the Right Thing on Mac but not other platforms).

**Comment by johann-petrak** [comment]
Ah ok, that  is it then -- I have 1.8 on that system. 
Does this mean 11 is the minimum recommended version for users to use then?

**Comment by ianroberts** [comment]
That's a very good question - @greenwoodma what do you think?  We definitely need to keep _building_ gate-core on Java 8 as long as we want to maintain compatibility with that version, but what should we recommend users use when _running_ it?

**Comment by johann-petrak** [comment]
Just told my Java installation to update and all I got is a more recent version of Java 8. Does not suggest or do any updating to a version beyond 8.
On the web, the first hit java.com suggests that the recommended version is 8_231

That more recent version of Java 8 actually shows both the installer and GUI larger and better sized, but it is still not a pretty sight, because it is essentially just a blown-up version of the tiny one, and thus looks blurred and ugly. 

The only alternative appears to be the more scary process of installing the latest JDK (13 atm) - after that, GATE looks really good. 
I believe another problem with the latest JDK is that it gets released frequently and users may have to update. There is a version 11 "LTS" release, but to download that, Oracle requires a login with a user id registered with Oracle!

**Comment by greenwoodma** [comment]
I was actually looking at this earlier in relation to #85.

Firstly, Oracle no longer offer a JDK under a license we can recommend. We updated the download page to recommend people use [AdoptOpenJDK](https://adoptopenjdk.net/). This will require a new download and install as there is no automatic upgrade path from an installed Oracle version; in fact I think the Oracle version only every suggested minor versions as updates so a 1.8 install wouldn't even suggest 1.9.

As for version, I'd certainly suggest we advise 1.9 or above as that's the first version to properly support HiDPI monitors which are now common enough to be a problem. Like Ian, I'd suggest we continue to build against 8 for now as that means we don't mess with any existing installs who want to upgrade GATE but can't update Java (NHS comes to mind).

To fix this and #85 I suggest we also advise that people pick a 64 bit version where at all possible.

I also tested GATE on 1.14ea yesterday and it works fine.

So my suggestion would come down to...

> _GATE Developer 9.0 works best with a 64 bit Java 9 or later JDK (Java 8 can be used if absolutely necessary). This is widely available from GNU/Linux package repositories. The AdoptOpenJDK website provides packages for various operating systems, and is particularly suitable for Windows users - ensure you enable the "JavaSoft (Oracle) registry keys" option in the installer, otherwise gate.exe will not work._
